cmd/dictpress/init.go

```diff
@@ -163,6 +163,7 @@ func initHTTPServer(app *App, ko *koanf.Koanf) *echo.Echo {
 	}
 
 	// Admin handlers and APIs.
+	a.GET("/api/entries/:fromLang/:toLang", handleSearch)
 	a.GET("/api/entries/:fromLang/:toLang/:q", handleSearch)
```

Comment:
Do we need the endpoint with query as part of path param if we can migrate to the proposed change ?

Reply:
Yeah the search query can be a GET param just like how `tag` is. This would need change in two routes. The search route can be made like this instead:

```bash
/api/dictionary/:fromLang/:toLang/search?q=keyword&tag=something&type=seomthing
/api/entries/:fromLang/:toLang/search?q=keyword&tag=something&type=seomthing
```

But this would require API migration and might break someone's workflow using the API. Making this change would be a lot more than what I have time for. But something to think about. Meanwhile can have this new endpoint added please?